### ISSUE TITLE
 yarn run build:win32 报错

### ISSUE THREAD

**Issue by 82318179** [LulumiProject/lulumi-browser#48]
yarn 没有问题
yarn run build:dll 没有问题
$ yarn run dev 没有问题
yarn run build:win32 报错
`
    ERROR in E:/lulumi/lulumi-browser-master/src/main/api/lulumi-extension.ts
    ERROR in E:/lulumi/lulumi-browser-master/src/main/api/lulumi-extension.ts(11,20):
    TS2306: File 'E:/lulumi/lulumi-browser-master/src/main/constants.ts' is not a module.

    ERROR in E:/lulumi/lulumi-browser-master/src/main/constants.ts
    ERROR in E:/lulumi/lulumi-browser-master/src/main/constants.ts(1,1):
    TS1085: Octal literals are not available when targeting ECMAScript 5 and higher. Use the syntax '0o0'.

    ERROR in E:/lulumi/lulumi-browser-master/src/main/index.ts
    ERROR in E:/lulumi/lulumi-browser-master/src/main/index.ts(23,23):
    TS2306: File 'E:/lulumi/lulumi-browser-master/src/main/constants.ts' is not a module.

    ERROR in E:/lulumi/lulumi-browser-master/src/main/constants.ts
    ERROR in E:/lulumi/lulumi-browser-master/src/main/constants.ts(1,41):
    eofline: file should end with a newline

    ERROR in E:/lulumi/lulumi-browser-master/src/main/constants.ts
    ERROR in E:/lulumi/lulumi-browser-master/src/main/constants.ts(1,41):
    semicolon: Missing semicolon

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

E:\lulumi\lulumi-browser-master>`

**Comment by qazbnm456** [comment]
Thanks for trying out! I'll look into it as soon as possible. 😄

**Comment by jiumengs** [comment]
我的也报错了 不过这个真的是一个伟大的项目

**Comment by qazbnm456** [comment]
@jiumengs Thanks for liking this project! I'm currently revamping the building process and moving the codebase to support Electron v9. You may revisit this project later to make your build succeed.

謝謝支持！我現在正在針對程式碼的構建及底層支援部分進行重寫以遷移至 Electron 版本 9。或許過些時日您們就可以再次重新進行構建，那時應該就不會再有問題了！ 一旦有新的進展我會再更新至這個 issue 中，謝謝。

**Comment by jiumengs** [comment]
很期待

**Comment by jiumengs** [comment]
@qazbnm456 
Is the new version electron version 9 considering using javascript instead of Typescript

新版本electron版本9是否考虑使用javascript而不再使用typescript

**Comment by qazbnm456** [comment]
@jiumengs No. It will still stick to TypeScript cause it provides better readability and security. There are no drawbacks to using TypeScript so far.

沒有打算換掉 TypeScript 喔！目前使用 TS 沒有任何壞處。

**Comment by jiumengs** [comment]
@qazbnm456 Well, I'm going to typescript  It's a sad story
那好吧 我要去看typescript了 真是一个悲伤的故事
